indico/modules/categories/templates/display/event_list.html

```diff
@@ -0,0 +1,158 @@
+                {% trans count=future_event_count -%}
+                    There is {{ count }} event in the future.
+                {%- pluralize -%}
+                    There are {{ count }} events in the future.
+                {%- endtrans %}
+            {% endcall %}
+        {% endif %}
+        <div>
+            {{ event_list_block(events_by_month, format_event_date, is_recent, happening_now) }}
+        </div>
+        {% if past_event_count %}
+            {% call hidden_events_block('past-events', category, past_events_by_month, format_event_date, is_recent,
+                                        happening_now, before=past_threshold, message_first=false) %}
+                {% trans count=past_event_count -%}
+                    There is {{ count }} event in the past.
+                {%- pluralize -%}
+                    There are {{ count }} events in the past.
+                {%- endtrans %}
+            {% endcall %}
+        {% endif %}
+    </div>
+    <script type="text/javascript">
+        var $eventList = $('.event-list');
+        var $futureEvents = $eventList.find('.future-events');
+        var $pastEvents = $eventList.find('.past-events');
+
+        setupToggleEventListButton($futureEvents);
+        setupToggleEventListButton($pastEvents);
+
+        function setupToggleEventListButton(wrapper) {
+            var $wrapper = $(wrapper);
+            var $content = $wrapper.find('.events');
+
+            function updateMessage(visible) {
+                $wrapper.find('.js-hide-message').toggle(visible);
+                $wrapper.find('.js-show-message').toggle(!visible);
+            }
+            updateMessage(!$content.is(':empty'));
+
+            $wrapper.find('.js-toggle-list').on('click', function(evt) {
+                evt.preventDefault();
+                if ($content.is(':empty')) {
+                    $.ajax({
+                        url: $content.data('event-list-url'),
+                        data: {
+                            before: $content.data('event-list-before'),
+                            after: $content.data('event-list-after')
+                        },
+                        error: handleAjaxError,
+                        success: function(data) {
+                            $content.html(data.html);
+                            $content.show();
+                            updateMessage(true);
+                        }
+                    });
+                } else if ($content.is(':visible')) {
+                    $content.hide();
+                    updateMessage(false);
+                } else {
+                    $content.show();
+                    updateMessage(true);
+                }
+            });
+        }
+    </script>
+{% endmacro %}
+
+{% macro event_list_block(events_by_month, format_event_date, is_recent, happening_now) %}
+    {% for month in events_by_month %}
+        <h4 {% if month.is_current %}class="currentMonth"{% endif %}>
```

Comment:
`<h4 class="{% if month.is_current %}currentMonth{% endif %}">` is more readable IMO (better when it comes to syntax highlighring)